src/compas_model/viewer/modelobject.py

```diff
@@ -0,0 +1,16 @@
+from compas_model.scene.modelobject import ModelObject as BaseModelObject
+from compas_viewer.scene import ViewerSceneObject
+
+
+class ModelObject(ViewerSceneObject, BaseModelObject):
+    def _read_points_data(self) -> None:
+        pass
+
+    def _read_lines_data(self) -> None:
+        pass
+
+    def _read_frontfaces_data(self) -> None:
+        pass
+
+    def _read_backfaces_data(self) -> None:
+        pass
```

Comment:
Currently these four functions must be implemented. Maybe we should simply allow to skip this in viewer, for simplicity.

Reply:
make it options